src/postgres-data/PsSource.ts

```diff
@@ -42,7 +42,7 @@ export default class PsSource implements StoreDataSourceStrategy {
     }
     if (state.manuscriptID && state.projectID) {
       this.data = await buildData(state.projectID, state.manuscriptID, this.api)
-      this.utilities = buildUtilities(() => this.data, this.api, setState)
+      this.utilities = buildUtilities(() => this.data, setState, this.api)
```

Comment:
I wonder what is the purpose of this change?

Reply:
Just wanted to make the `api` param at the end to make it consistent with the other call. Not necessary though.